lib/run_loop/device.rb

```diff
@@ -750,7 +750,11 @@ def simulator_running_user_app_pids
     # @!visibility private
     def simulator_running_system_app_pids
       base_dir = xcode.developer_dir
-      sim_apps_dir = "Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/Applications"
+      if xcode.version_gte_90?
+        sim_apps_dir = "Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/Applications"
+      else
+        sim_apps_dir = "Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/Applications"
+      end
```

Comment:
@polyakrecords Can you test this against Xcode 9.0 on your machine?

```
$ git fetch
$ git checkout -t feature/xcode-9.3-support
$ DEVELOPER_DIR=/Xcode/9.0/Xcode.app/Contents/Developer bundle exec rspec spec/integration/device_spec.rb:15
```

Thanks!

Reply:
@jmoody Tested. All green.